### PR TITLE
I have given answers

### PR DESCRIPTION
**Pull request by MemoonaMobeen**
**##** What type of PR is this? (check all applicable)

-  [Memoona Mobeen ] 🚀 Added Name
- [ ] ✨ Feature
- [yes ] 🌟 stared the repo
- [no ] 🐛 Grammatical Error
- [yes ] 📝 Documentation Update
- [ ] 🚩 Other

## Description

I ve updated this

## Add Link of GitHub Profile

https://github.com/MemoonaMobeen

### CODE REVIEW

**Review by AndroidLion48 — DISMISSED** [review]

File: Languages/HTML5/questions.json [hunk 1]
```diff
@@ -2,30 +2,19 @@
...
       "id": 0,
-      "question": "Please input a "What Is" question relevant to your language here.",
+      "question": "Are the HTML tags and elements the same thing? ...",
       "details": "Expanded information on the correct answer to the question goes here.",
       "questionType": "multi",
       "trueOrFalse": false,
       "shortAns": "Please input a short ''It is'' Ans to the question above here that ends with punctuation.",
-      "tag": "Your qestion's subject Here"
+      "tag": "HTML 5"
     },
```
[the remark this answers is not in the document]
Comment (MemoonaMobeen) [reply Languages/HTML5/questions.json:11]: okay, Got it @Tc2r1. I'm going to Add 3 Quiz Answers to JSON.

**Comment by MemoonaMobeen** [comment]
Hey @Tc2r1, I ve addes 3 anwers to JSON

**Review by Tc2r1 — APPROVED** [review]
All of this looks really great, Omega Good job!

@MemoonaMobeen  would you be interested in making a webpage quiz version of the app?

**Comment by MemoonaMobeen** [comment]
hey @Tc2r1 firstly plz accept my PRs for **Hactoberfest**
As for quiz app sure!

**Comment by MemoonaMobeen** [comment]
hey @Tc2r1 firstly plz accept my PRs for **Hactoberfest**
As for quiz app sure!

**Comment by MemoonaMobeen** [comment]
> All of this looks really great, Omega Good job!
> 
> @MemoonaMobeen would you be interested in making a webpage quiz version of the app?
> 
> (Please don't forget to ⭐ the repo! Thanks (top right corner lol)

Thankx @Tc2r1 kindly acccept this for hactoberfest

**Comment by Tc2r1** [comment]
@MemoonaMobeen 

Hi, so currently the application is a project consisting of a bunch of modules that all build the same application using different frameworks/libraries.  Currently they all build to show an android Quiz by pulling the information from the languages/android folder. 

There are a few different ways you can contribute to the project.
1) Add a screen that allows you to pick different types of quizzes and flesh out the repository layer to handle loading different types of quizzes. 

2) Make splash screens for one of the pre-existing modules

3) Copy a template module (or in the case that you want to make the app in something besides native Kotlin or android, use the template as reference) and recreate the application in your own image using your own selection of libraries, (Like Hilt for example) 

There is documentation for these options in the original readme for the project. 

To build the project, after you have the project on your local computer, open the `Interview_Prep_App` folder in Android studio  and it will compile the project, then select whichever module you'll work in and go from there. 

All Modules share the resources from the sharedResources Module.  If you have any questions please post them in discussions or an issue. 

**For those who are familiar with other tech-stacks. We would love for you to recreate the application as a website/web-app/react-native/kMM/python app, nodes.js, etc  By doing so you gain experience and you show new developers how to create a RESTful experience.** 

Please add your idea on how you'd like to recreate the app as a new issue :) 

What do you think?

**Comment by MemoonaMobeen** [comment]
Hey, @Tc2r1 plz accept this for hacktoberfest

**Pull request merged**